src/main/java/org/humancellatlas/ingest/security/AccountRepository.java

```diff
@@ -12,4 +12,5 @@
 
     Account findByProviderReference(String providerReference);
 
+    List<Account> findAccountByRolesEquals(Role role);
```

Comment:
1. Why `Roles` in plural? 
2. Why is `Equals` necessary?

Reply:
1. Because the property is defined as `roles` in `Account.java`: https://github.com/ebi-ait/ingest-core/blob/c19ccf17f62ef5fd9e7fb9221c136e98c5a7b8b1/src/main/java/org/humancellatlas/ingest/security/Account.java#L60
Using just `Role` gives this error
`Caused by: org.springframework.data.mapping.PropertyReferenceException: No property role found for type Account! Did you mean 'roles'?`
2. It isn't, just removed it